en/airframes/adding_a_new_frame.md

```diff
@@ -127,7 +127,7 @@ Where each number from left to right means:
 * S: Indicates the first input scaler: It takes input from control group #0 (Flight Control) and the first input (roll). It scales the roll control input * 0.6 and reverts the sign (-0.6 becomes -6000 in scaled units). It applies no offset (0) and outputs to the full range (-1..+1)
 * S: Indicates the second input scaler: It takes input from control group #0 (Flight Control) and the second input (pitch). It scales the pitch control input * 0.65. It applies no offset (0) and outputs to the full range (-1..+1)
 
-> **Note** In short, the output of this mixer would be SERVO = ( (roll input * -0.6 + 0) + (pitch input * 0.65 + 0) ) * 1 + 0
+> **Note** In short, the output of this mixer would be SERVO = ( (roll input \* -0.6 + 0)  \* 1 + (pitch input \* 0.65 + 0) ) \* 1 + 0
```

Comment:
If you add the scaling for roll you need to add it for pitch as well:
```suggestion
> **Note** In short, the output of this mixer would be SERVO = ( (roll input \* -0.6 + 0)  \* 1 + (pitch input \* 0.65 + 0)  \* 1 ) \* 1 + 0
```

Reply:
> If you add the scaling for roll you need to add it for pitch as well:

@bkueng My point was that pitch already had the scaling but roll did not - so I added it to roll. So doesn't this add the `x1` scaling for pitch a second time?

Reply:
No, the existing `\* 1` scaling applies to both roll and pitch (there is another bracket). It refers to the output scaling.
The added one refers to the second (output) scaling of the input scaler.

Reply:
Thanks @bkueng . I accepted this. It did take me a while to work out what you are talking about. Might have been easier to parse if some of the scalers here were not 1 - ie so that we could easily visually map between roll, pitch input scalers, and  0: output scaler.